app/views/govuk_component/analytics_meta_tags.raw.html.erb

```diff
@@ -2,19 +2,27 @@
   # We have to call deep_symbolize_keys because we're often dealing with a
   # parsed JSON document which will have string keys by default, but our
   # components use symbol keys and we want consistency.
-  links_hash = content_item.to_h.deep_symbolize_keys[:links] || {}
+  content_item_hash = content_item.to_h.deep_symbolize_keys
+  links_hash = content_item_hash[:links] || {}
```

Comment:
The distinction is a bit fuzzy at the moment, but inside the "presenter" bit (eg, the complex processing of `content_item`) we can use any ERB features, as that is GOV.UK specific. We should avoid them in the "view" bit, which should be translatable in to other (simpler) languages, like mustache. The more ERB features we use in "views" the harder it is, as [mentioned before](https://github.com/alphagov/static/pull/669#discussion_r42742005).

As both the "presenter" and view" are in the same file, and we don't transpile the "views" yet, this is a bit obtuse. Sorry. Hopefully this will be covered better in some [component principles documentation](https://trello.com/c/ZyhODRnG/19-document-component-principles), when I have time to write it :(

In the mean time, `fetch` isn't a big deal in either place, as it's semantically equivalent to `hash[:key] || "default"`, right? So both are translatable. We've avoided `fetch` in (most) other view templates to avoid encouraging use of more complex usage, which should hopefully be bettered covered by the principles, when we have them.

So i'd be fine with `fetch` being used like this, but we should avoid using it in the view, or other "view only" components (which is most of them). 

Does that make sense?